packages/Config/RectorConfig.php

```diff
@@ -69,22 +69,12 @@ public function skip(array $criteria): void
         $parameters->set(Option::SKIP, $criteria);
     }
 
-    public function importNames(): void
+    public function importNames(bool $disableImportDocBlockNames = false): void
```

Comment:
The value should be inverted, to avoid double neagive mix.


```diff
-disableImport = false
+import = true
```

Reply:
I did a bit more refactoring to make the `importNames` method call more clear and not confusing.

Reply:
@TomasVotruba Check please this PR https://github.com/rectorphp/rector-nette/pull/76

Reply:
CI green again 🌲